script/hardhat/fork/vip-diamond-comptroller-mainnet.ts

```diff
@@ -0,0 +1,363 @@
+import { impersonateAccount } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { VBep20 } from "../../../typechain";
+import { cutParams as params } from "../../deploy/comptroller/cut-params.json";
+import { forking, pretendExecutingVip, testVip } from "./vip-framework";
+import { ProposalType } from "./vip-framework/types";
+import { makeProposal } from "./vip-framework/utils";
+
+const UNITROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+
+const DIAMOND_CUT_FACET = "";
+const DIAMOND = "";
+const DIAMOND_INIT = "";
+const cutParams = params;
+
+export const vipDiamond = () => {
+  const meta = {
+    version: "v1",
```

Comment:
This is a schema version we are proposing; it should be v2. @kkirka, am I correct? Could you please brief here?

Reply:
VIP is moved to vips repo.